playbooks/management/reading-list.md

```diff
@@ -31,6 +31,10 @@ recommend it!_
 - Radical Candor (Kim Scott, 2017)
   - (From [Sarah][sarah]) Another book on communication, this time about how to care about people and challenge
     them at the same time through candid feedback.
+- Multiplier (Liz Wiseman, 2010)
+  - (From [Adam][adam]) This was our another book from the manager book club at Artsy. Describes the behaviors of
```

Comment:
I think you have to add your name below for the link to fill in! But otherwise this looks great-- ty for adding and feel free to merge when ready. 😄

Reply:
Thank goodness I PR'ed this 😅 Thanks for teaching me how that works!